sentry_streams/sentry_streams/adapters/arroyo/reduce.py

```diff
@@ -90,7 +90,7 @@ def add(self, value: Any) -> Self:
                 self.offsets[partition] = max(offsets[partition], self.offsets[partition])
 
             else:
-                self.offsets.update(offsets)
+                self.offsets[partition] = offsets[partition]
```

Comment:
?

Reply:
If you expand the code snippet, there was a bug, that got fixed here